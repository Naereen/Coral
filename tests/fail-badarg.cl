x == 2.0

b = 4.0

if x != 7.3:
	b = 7.2

print(4)

a = 1 + b

print(7)
