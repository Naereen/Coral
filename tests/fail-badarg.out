4
RuntimeError: invalid argument types for + operator
